src/grid/onedgrid.py

```diff
@@ -114,20 +114,24 @@ def GaussChebyshev(npoints):
     return OneDGrid(points[::-1], weights)
 
 
-def HortonLinear(npoints):
-    """Generate 1D grid on [0, npoints] interval using equally spaced uniform distribution.
+def HortonLinear(start, stop, step):
```

Comment:
There is a little problem with start stop idea. Horton style transformation maps 0 to rmin and N-1 value to rmax.

Reply:
Agreed. The original may seem limited but there is little benefit in extending it since this may become deprecated in future.

Reply:
I would still recommend keep HortonLinear in the old form. The start and end other than `0` and `n-1` will be mapped out of proper domain.

Reply:
Ok to keep the old style for now. The unit tests would need an additional hacky line to also shift the domain by `+1`.

Reply:
Removed the commit changing `HortonLinear`, so it's back to the old form.